projects/core/src/product/occ/product-search.service.spec.ts

```diff
@@ -5,7 +5,10 @@ import {
 } from '@angular/common/http/testing';
 
 import { OccProductSearchService } from './product-search.service';
-import { OccConfig, SearchConfig } from '@spartacus/core';
+import {
+  OccConfig,
+  SearchConfig
+} from 'projects/backend/occ-client/lib/models';
```

Comment:
Split the two imports and reference the actual place they are from in core. SeachConfig is in product and OccConfig is in occ. ex: import { OccConfig } from '../../occ'; . You could import both from public_api but I still think it's less clean.